CMakeLists.txt

```diff
@@ -98,8 +98,6 @@ if(HASH_JOIN)
   set(CUDA_NVCC_FLAGS  ${CUDA_NVCC_FLAGS};-DHASH_JOIN)
 endif()
 
-find_package(CUDA)
-
```

Comment:
This doesn't seem correct to me.

Reply:
Is duplicated, see lines above

Reply:
That line was repeated without reason that is why i removed

Reply:
Oh yes, I see now. Sorry for the noise.

Reply:
No problem, thanks for taking the time to review this stuff ;)